barretenberg/cpp/src/barretenberg/proof_system/circuit_builder/ultra_circuit_builder.test.cpp

```diff
@@ -712,6 +712,40 @@ TEST(ultra_circuit_constructor, rom)
     EXPECT_EQ(result, true);
 }
 
+/**
+ * @brief A simple-as-possible RAM read test, for easier debugging
+ *
+ */
+TEST(ultra_circuit_constructor, ram_simple)
+{
+    UltraCircuitBuilder builder;
+
+    // Initialize a length-1 RAM array with a single value
```

Comment:
length=1 ?

Reply:
I meant it to be a hyphen but I see now that it could be read as length minus 1. Will change